components/match2/wikis/fifa/match_summary.lua

```diff
@@ -190,8 +191,8 @@ function CustomMatchSummary._players(players, opponentIndex, winner)
 		:css('padding', '2px 8px')
 		:css('text-align', flip and 'right' or 'left')
 		:css('width', '35%')
-		:node(OpponentDisplay.BlockPlayers{
-			opponent = {players = players},
+		:node(OpponentDisplay.BlockOpponent{
+			opponent = {players = players, type = Opponent.solo},
```

Comment:
this was already thinking about team matches on that wiki where there might be >1 players in `players`
(even >4 was possible iirc)